assign_nodes.py

```diff
@@ -64,6 +64,16 @@ def search_nodes(nodes, kg, examples):
 				print(found_nodes.iloc[0:nrow,].to_string())
 				user_input = input("Input node'label': ")
 				if node_in_search(found_nodes,user_input):
+					#Manage if there are 2 duplicate label names for both pkl and kg-covid19
+					if len(found_nodes[found_nodes['label'] == user_input][['label','entity_uri']]) > 1:
+						user_input = input("Input node 'entity_uri': ")
+						if node_id_in_search(found_nodes,user_input):
+							node_label = kg.labels_all.loc[kg.labels_all['entity_uri'] == user_input,'label'].values[0]
```

Comment:
Need to edit this to search for 'id' column of labels_all instead, and ensure that column names are identical between pkl and kg-covid19 graphs

Reply:
I think this is something we fixed in the fall, but now can't remember. Thanks for checking.

Reply:
I think this was identified as a conflict between main and this branch. I resolved the conflict to search for 'id'.

Reply:
Thanks for catching. I'm a bit confused because the process_pkl_files function still creates a labels df with the "entity_uri" column. Did we just somehow get out of sync here?

Reply:
Need to switch all code to use "entity_uri" rather than "id" for node identifiers, was using "id" in other workflows but that isn't necessary here.